src/rgw/driver/d4n/d4n_policy.cc

```diff
@@ -54,8 +54,8 @@ int LFUDAPolicy::init(CephContext *cct, const DoutPrefixProvider* dpp, asio::io_
   static auto obj_callback = [this](
           const DoutPrefixProvider* dpp, std::string& key, std::string version, bool dirty, uint64_t size, 
 			    time_t creationTime, const rgw_user user, std::string& etag, const std::string& bucket_name, const std::string& bucket_id,
-			    const rgw_obj_key& obj_key, optional_yield y) {
-    update_dirty_object(dpp, key, version, dirty, size, creationTime, user, etag, bucket_name, bucket_id, obj_key, y);
+			    const rgw_obj_key& obj_key, optional_yield y, std::string& restore_val) {
```

Comment:
this must not show up as a diff here - you need to rebase your branch

Reply:
I rebased today but it is still showing up. I'm confused; isn't this a correct change?

Reply:
If its added by you then its ok.